components/giftbox.vue

```diff
@@ -0,0 +1,280 @@
+              { selected: feature.uid === selectedFeature.uid },
+              { seen: seenFeatures.has(feature.uid) },
+            ]"
+            @click="switchFeature(feature.uid)"
+          >
+            <div class="notification-icon-wrapper">
+              <fa
+                v-if="!seenFeatures.has(feature.uid)"
+                icon="eye"
+                class="eye-icon"
+              />
+              <fa v-else icon="eye-slash" class="eye-icon" />
+            </div>
+            <p>{{ feature.title }}</p>
+          </div>
+        </div>
+        <div v-if="selectedFeature" class="features-detail">
+          <h6>{{ selectedFeature.title }}</h6>
+          <a
+            v-if="selectedFeature.image"
+            :href="selectedFeature.image"
+            target="_blank"
+          >
+            <img
+              :src="selectedFeature.image"
+              :alt="selectedFeature.title"
+              class="feature-image"
+            />
+          </a>
+          <!-- eslint-disable-next-line vue/no-v-html -->
+          <div class="feature-content" v-html="selectedFeature.body" />
+          <div class="flex-right">
+            <a
+              v-if="selectedFeature.callToAction"
+              :href="selectedFeature.callToAction"
+              target="_blank"
+            >
+              <b-button class="btn-call-to-action" squared variant="primary">
+                {{ selectedFeature.callToActionLabel }}
+              </b-button>
+            </a>
+          </div>
+        </div>
+        <div v-else class="features-detail">
+          <p>{{ $t('giftbox.no_features') }}</p>
+        </div>
+      </div>
+    </b-modal>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'Giftbox',
+    data: () => ({
+      loading: true,
+      features: [],
+      selectedFeature: undefined,
+      seenFeatures: new Set(),
+    }),
+    created() {
+      this.fetchFeatures().then((res) => {
+        this.features = res;
+        if (this.features.length > 0) {
+          this.selectedFeature = this.features[0];
+          this.seenFeatures.add(this.selectedFeature.uid);
+        }
+      });
+    },
+    methods: {
+      async fetchFeatures() {
+        this.loading = true;
+        const response = await fetch(
+          process.env.newFeaturesUrl
+        );
+        const { data } = await response.json();
+        this.loading = false;
+        return data;
+      },
+      showModal() {
+        if (!this.loading) {
+          this.$refs['giftbox-modal'].show();
+        }
+      },
+      hideModal() {
+        this.$refs['giftbox-modal'].hide();
+      },
+      switchFeature(uid) {
+        if (uid) {
+          this.selectedFeature = this.features.find(
+            (feature) => feature.uid === uid,
+          );
+          this.seenFeatures.add(this.selectedFeature.uid);
+        }
+      },
+    },
+  };
+</script>
+
+<style lang="scss" scoped>
+  @import '../assets/styles/_colors';
```

Comment:
Better to import via the `styleResources` setting in `nuxt.config.js` see [https://dev.to/ceppeu/using-sass-global-variables-in-nuxt-js-j0k](https://dev.to/ceppeu/using-sass-global-variables-in-nuxt-js-j0k)

Maybe you could make a `main.scss` and then import it in the `nuxt.config.js`